### PR TITLE
Fix README to be more accurate & use FACOLS test setup

### PR DESCRIPTION
**Pull request by kahlouie**
Connects #5656

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,9 +138,16 @@ docker-compose down -v
 rake [RAILS_ENV=<test|development|stubbed>] db:setup
 rake [RAILS_ENV=<test|development|stubbed>] db:seed
 
-# setup local VACOLS (FACOLS)
-rake [RAILS_ENV=<test|development>] local:vacols:setup
-rake local:vacols:seed
+# setup local test VACOLS (FACOLS)
+docker ps
+# Get the CONTAINER ID of the container named VACOLS_DB-development
+docker stop <CONTAINER ID>
```
Comment (sansb) [inline README.md]: Worth noting that these steps will only be necessary if you ran the project before we switched to having two containers.

File: README.md [hunk 2]
```diff
@@ -30,10 +32,6 @@ Then run the following:
 
 *NOTE* If when running `gem install bundler` above you get a permissions error, this means you have not propertly configured your rbenv. Do not proceed by running `sudo gem install bundler`.
 
-You need to have Chromedriver running to run the Capybara tests. Let `brew` tell you how to do that:
-
-    brew info chromedriver
-
```
Comment (shanear) [inline README.md]: nice 👌

File: README.md [hunk 3]
```diff
@@ -253,6 +258,16 @@ what roles they have and therefore what pages they can access. To add new users
...
 should be a comma separated list of roles you want that user to have.
 
+To use intake features as the users, you'll need to toggle two features in a
+rails console `rails c`:
+
+```
+[1] FeatureToggle.enable!(:intakeAma)
+=> true
+[2] FeatureToggle.enable!(:intake)
+=> true
```
Comment (shanear) [inline README.md]: This is making me realize we should probably find a better solution for permissions than our feature flag

File: README.md (hunk 1 above)
Comment (shanear) [reply README.md]: If that's the case, we may just want to leave these steps out

Comment (kahlouie) [reply README.md]: I think that'll always be the case unless you only need development up and you aren't planning on running tests. I can add a comment that says this.

Comment (sansb) [reply README.md]: Could you explain further?

**Pull request merged**